scripts/yrd/uninstall

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
See how ipfs/uninstall does it. Should disable, then stop, then daemon-reload, then remove. Also notice the `|| true` and `rm -f` flag. These prevent errors when things aren't running to being with when you uninstall.

```
# Uninstall IPFS
sudo systemctl disable ipfs.service 2>/dev/null || true
sudo systemctl stop ipfs.service 2>/dev/null || true
sudo systemctl daemon-reload
sudo rm -f /usr/local/bin/ipfs
sudo rm -f /etc/systemd/system/ipfs.service
```